frontend/src/scenes/plugins/edit/CapabilitiesInfo.ts

```diff
@@ -0,0 +1,10 @@
+export const capabilitiesInfo: Record<string, string> = {
+    processEvent: 'Used to read and modify events before ingestion',
+    processEventBatch: 'Used to read and modify and modify events in batches before ingestion',
```

Comment:
Maybe "Provides" here too instead of "Used to"? I dunno why, but "Used to" doesn't fit this IMO, it sounds to me somewhat… passive?

Reply:
I played around with the wording quite a bit here - never got to something I was super happy with. Will try again.